test/test_front_matter_defaults.rb

```diff
@@ -27,6 +27,30 @@ class TestFrontMatterDefaults < JekyllUnitTest
     end
   end
 
+  context "A site with full front matter defaults (glob)" do
+    setup do
+      @site = fixture_site({
+        "defaults" => [{
+          "scope"  => {
+            "path" => "contacts/*.html",
```

Comment:
That's what would happen with the current code, yeah. I'm wondering if it makes sense to have `*.html` apply to all files in my entire source that end in `.html`. If I were using globs, that's what I would expect.